sdk/communication/Azure.Communication.Calling.Server/tests/ConversationClients/ConversationClientsTests.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+
+namespace Azure.Communication.Calling.Server.Tests
+{
+    public class ConversationClientsTests
+    {
+        [TestCaseSource(nameof(TestData_StartRecording))]
+        public async Task StartRecording_Passes(string expectedConversationId, Uri expectedCallbackUri)
```

Comment:
I don't think that test is actually testing anything, right? It only verifies that you are passing the same values.
You should avoid mocking the object or class you want to actually test, you probably want to mock the Http call instead to let the runner test your code functionality.
You can accomplish that using this as part of your `ConversationClient` creation:
```c
var mockResponse = new MockResponse(statusCode);
mockResponse.SetContent(<your expected response>);

var conversationClientOptions = new CallClientOptions
{
        Transport = new MockTransport(mockResponse)
};

var convClient = new ConversationClient(uri, communicationTokenCredential, conversationClientOptions);
```

Reply:
Thanks for the comments!
We will address this comments later.
To get this PR checked in, we are agreed with the approvers that not focusing on test related comments.